aries-site/src/layouts/main/Layout.js

```diff
@@ -52,16 +60,41 @@ export const Layout = ({
   } = getPageDetails(titleProp);
   const layout = isLanding ? 'plain' : pageLayout;
 
+  const defaultFeedback = {
+    'like-rating': '',
+    'star-rating': 0,
+    'text-area': '',
+  };
+
   const MainContentWrapper = isLanding ? Fragment : PageContent;
-  const size = useContext(ResponsiveContext);
+  const breakpoint = useContext(ResponsiveContext);
+  const [open, setOpen] = useState(false);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+  const [value, setValue] = useState(defaultFeedback);
+  const [isSucessful, setIsSuccess] = useState(false);
+
+  const close = () => {
```

Comment:
```suggestion
  const closeFeedbackModal = () => {
```
something a little more descriptive